workflows/targeted.nf

```diff
@@ -0,0 +1,693 @@
+            hmf_data.segment_mappability,
+            panel_data.driver_gene_panel,
+            hmf_data.ensembl_data_resources,
+        )
+
+        ch_versions = ch_versions.mix(SAGE_CALLING.out.versions)
+
+        ch_sage_germline_vcf_out = ch_sage_germline_vcf_out.mix(SAGE_CALLING.out.germline_vcf)
+        ch_sage_somatic_vcf_out = ch_sage_somatic_vcf_out.mix(SAGE_CALLING.out.somatic_vcf)
+        ch_sage_germline_dir_out = ch_sage_germline_dir_out.mix(SAGE_CALLING.out.germline_dir)
+        ch_sage_somatic_dir_out = ch_sage_somatic_dir_out.mix(SAGE_CALLING.out.somatic_dir)
+
+    } else {
+
+        ch_sage_germline_vcf_out = ch_inputs.map { meta -> [meta, [], []] }
+        ch_sage_somatic_vcf_out = ch_inputs.map { meta -> [meta, [], []] }
+        ch_sage_germline_dir_out = ch_inputs.map { meta -> [meta, []] }
+        ch_sage_somatic_dir_out = ch_inputs.map { meta -> [meta, []] }
+
+    }
+
+    //
+    // SUBWORKFLOW: Annotate variants with PAVE
+    //
+    // channel: [ meta, pave_vcf ]
+    ch_pave_germline_out = Channel.empty()
+    ch_pave_somatic_out = Channel.empty()
+    if (run_config.stages.pave) {
+
+        PAVE_ANNOTATION(
+            ch_inputs,
+            ch_sage_germline_vcf_out,
+            ch_sage_somatic_vcf_out,
+            ref_data.genome_fasta,
+            ref_data.genome_version,
+            ref_data.genome_fai,
+            hmf_data.sage_pon,
+            panel_data.pon_artefacts,
+            hmf_data.sage_blocklist_regions,
+            hmf_data.sage_blocklist_sites,
+            hmf_data.clinvar_annotations,
+            hmf_data.segment_mappability,
+            panel_data.driver_gene_panel,
+            hmf_data.ensembl_data_resources,
+            hmf_data.gnomad_resource,
+        )
+
+        ch_versions = ch_versions.mix(PAVE_ANNOTATION.out.versions)
+
+        ch_pave_germline_out = ch_pave_germline_out.mix(PAVE_ANNOTATION.out.germline)
+        ch_pave_somatic_out = ch_pave_somatic_out.mix(PAVE_ANNOTATION.out.somatic)
+
+    } else {
+
+        ch_pave_germline_out = ch_inputs.map { meta -> [meta, []] }
+        ch_pave_somatic_out = ch_inputs.map { meta -> [meta, []] }
+
+    }
+
+    //
+    // SUBWORKFLOW: Call CNVs, infer purity and ploidy, and recover low quality SVs with PURPLE
+    //
+    // channel: [ meta, purple_dir ]
+    ch_purple_out = Channel.empty()
+    if (run_config.stages.purple) {
+
+        PURPLE_CALLING(
+            ch_inputs,
+            ch_amber_out,
+            ch_cobalt_out,
+            ch_pave_somatic_out,
+            ch_pave_germline_out,
+            ch_gripss_somatic_out,
+            ch_gripss_germline_out,
+            ch_gripss_somatic_unfiltered_out,
+            ref_data.genome_fasta,
+            ref_data.genome_version,
+            ref_data.genome_fai,
+            ref_data.genome_dict,
+            hmf_data.gc_profile,
+            hmf_data.sage_known_hotspots_somatic,
+            [],  // sage_known_hotspots_germline
```

Comment:
Possibly minor, but the order for this was germline then somatic in SAGE_CALLING

Reply:
Good catch, even though minor I have changed argument ordering in SAGE_CALLING subworkflow invoke to be more consistent in https://github.com/nf-core/oncoanalyser/pull/64.

Generally, I've planned to reorder code after the release so that somatic variables/calls/blocks appears before germline - this isn't critical to functionality or the release but I also like to have things well structured!